packages/react-reconciler/src/ReactFiberLazyComponent.js

```diff
@@ -12,6 +12,21 @@ import type {LazyComponent, Thenable} from 'shared/ReactLazyComponent';
 import {Resolved, Rejected, Pending} from 'shared/ReactLazyComponent';
 import warning from 'shared/warning';
 
+export function resolveDefaultProps(Component, baseProps) {
```

Comment:
I think this is not just used for lazy.

Reply:
Even if it was this would be the wrong place for it.

Reply:
I don’t like shared. I prefer copying to putting things in shared. Makes it impossible to special case and encourages abstracting.

Reply:
So where do you want it? I guess I can create a file in the reconciler package but I don’t see why it deserves that when conceptually it’s related to our design for lazy.

Reply:
I kept it in `LazyComponent` for now because I don't want to duplicate it in reconciler bundle twice.